iroh-relay/src/server.rs

```diff
@@ -122,6 +138,33 @@ pub struct StunConfig {
     pub bind_addr: SocketAddr,
 }
 
+/// Configuration for the QUIC server.
+// TODO(ramfox): limits? Limiting connections or accepted # of connections from a single IP? from a single node ID?
```

Comment:
we also need to likely adjust the stream/connection limits for a server here

Reply:
https://github.com/n0-computer/iroh/issues/2967

Reply:
I took the liberty to edit that issue.